docs/background_info/meta_attributes.md

```diff
@@ -0,0 +1,96 @@
+> To a metaelement metaattributes could be compositional assigned (class "MetaAttribut"). The metaattributes are neither defined by the language INTERLIS nor by the meta model. They exist that the information that exceed INTERLIS could still be a component of the model data. The metaattributes has a name (attribute "Name"), that have to be unique among the the metaattributes of this metaelement, and it has a value (attribute: "Value").
+
+Translated from source: [STAN_d_DEF_2011-06-22_eCH-0117 Meta-Attribute für INTERLIS-Modelle.pdf](https://www.ech.ch/alfresco/s/ech/download?nodeid=788eb38a-bf2b-4f3d-96a8-addc37bba41f) (German) from eCH www.interlis.ch
+
+## Meta Attributes in Interlis Files
+
+### Comment vs. Meta Attribute
+
+An Interlis comment starts with `!!` and ends with a line end. A meta attribute starts with `!!` as well but followed by an `@`:
+
+### Syntax
+
+After the start of `!!@` the meta attribute name follows, then equal `=`, then the attribute value:
+
+```
+!!@<name>=<value>
+```
+Followed by the referenced element (MODEL, TOPIC, CLASS etc.)
+
+### Example
+
+ExceptionalLoadsRoute.ili:
+```
+!!@ furtherInformation=https://www.astra.admin.ch/
+MODEL ExceptionalLoadsCatalogues_V1 (en)
+AT "http://models.geo.admin.ch/ASTRA/"
+VERSION "2017-02-08"  =
+    IMPORTS CatalogueObjects_V1,Units;
+    !!@ topicInformation="Route-Type"
+    TOPIC TypeOfRouteCatalogue
+    EXTENDS CatalogueObjects_V1.Catalogues =
+    !!@ dispExpression="CONCAT(type, ' sometext')"
+    CLASS TypeOfRoute
+    EXTENDS CatalogueObjects_V1.Catalogues.Item =
+```
+
+`furtherInformation` is referenced to `ExceptionalLoadsCatalogues_V1`, `topicInformation` to `TypeOfRouteCatalogue` and `dispExpression` to `TypeOfRoute`.
+
+For more complex usage see the [specification of the Verein eCH](https://www.ech.ch/alfresco/s/ech/download?nodeid=788eb38a-bf2b-4f3d-96a8-addc37bba41f).
+
```

Comment:
The link does not work - when I look it up on https://ech.ch/de/standards/60625 I find this link:
https://ech.ch/index.php/de/dokument/1f67b229-f6bc-4fb4-a437-f91602e0b01d